src/worker/mod.rs

```diff
@@ -30,9 +36,15 @@ pub struct WorkerConfig {
     #[builder(default = "100")]
     pub max_outstanding_activities: usize,
     /// Maximum number of concurrent poll workflow task requests we will perform at a time on this
-    /// worker's task queue
+    /// worker's task queue. See also [nonsticky_to_sticky_poll_ratio]. A zero value will be treated
+    /// as if set to `1`.
```

Comment:
The minimum in case sticky is on is 2 according to the implementation below.
We should just return an error if passed less than 2 concurrent polls with sticky turned on and 1 with sticky off.

Reply:
Yeah I'm fine with just returning an error. Will do that.